zio-sbt-ci/src/main/scala/zio/sbt/ZioSbtCiPlugin.scala

```diff
@@ -252,50 +252,51 @@ object ZioSbtCiPlugin extends AutoPlugin {
             SetupLibuv,
             SetupJava("${{ matrix.java }}"),
             CacheDependencies,
-            checkout,
+            checkout
+          ) ++ (
             if (javaPlatformMatrix.values.toSet.isEmpty) {
-              Step.SingleStep(
-                name = "Test",
-                run = Some(prefixJobs + "sbt ${{ matrix.scala-project }}/test")
+              Seq(
+                Step.SingleStep(
+                  name = "Test",
+                  run = Some(prefixJobs + "sbt ${{ matrix.scala-project }}/test")
+                )
               )
             } else {
-              Step.StepSequence(
-                Seq(
-                  Step.SingleStep(
-                    name = "Java 11 Tests",
-                    condition = Some(Condition.Expression("matrix.java == '11'")),
-                    run = Some(
-                      prefixJobs + "sbt ${{ matrix.scala-project-java11 }}/test"
-                    )
-                  ),
-                  Step.SingleStep(
-                    name = "Java 17 Tests",
-                    condition = Some(Condition.Expression("matrix.java == '17'")),
-                    run = Some(
-                      prefixJobs + "sbt ${{ matrix.scala-project-java17 }}/test"
-                    )
-                  ),
-                  Step.SingleStep(
-                    name = "Java 21 Tests",
-                    condition = Some(Condition.Expression("matrix.java == '21'")),
-                    run = Some(
-                      prefixJobs + "sbt ${{ matrix.scala-project-java21 }}/test"
-                    )
+              Seq(
+                Step.SingleStep(
+                  name = "Java 11 Tests",
+                  `if` = Some(Condition.Expression("matrix.java == '11'")),
+                  run = Some(
+                    prefixJobs + "sbt ${{ matrix.scala-project-java11 }}/test"
+                  )
+                ),
+                Step.SingleStep(
+                  name = "Java 17 Tests",
+                  `if` = Some(Condition.Expression("matrix.java == '17'")),
+                  run = Some(
+                    prefixJobs + "sbt ${{ matrix.scala-project-java17 }}/test"
+                  )
+                ),
+                Step.SingleStep(
+                  name = "Java 21 Tests",
+                  `if` = Some(Condition.Expression("matrix.java == '21'")),
+                  run = Some(
+                    prefixJobs + "sbt ${{ matrix.scala-project-java21 }}/test"
                   )
                 )
               )
-
             }
           )
       )
 
     val DefaultTestStrategy =
       Job(
-        id = "test",
```

Comment:
Why the `id` is removed?

Reply:
The name is used, transformed to an id. This way we do not have to exclude the id field from the json. Do you prefer the explicit id field?